python-package/lightgbm/basic.py

```diff
@@ -1071,6 +1114,7 @@ def __init__(self, data, label=None, reference=None,
             For example, if you have a 100-document dataset with ``group = [10, 20, 40, 10, 10, 10]``, that means that you have 6 groups,
             where the first 10 records are in the first group, records 11-30 are in the second group, records 31-70 are in the third group, etc.
         init_score : list, numpy 1-D array, pandas Series or None, optional (default=None)
+            For multiclass classification can also be list of lists, numpy 2-D array or pandas DataFrame.
```

Comment:
This should be written after `init_score :` in allowed types description.
I guess something like this will be OK...
```suggestion
        init_score : list, list of lists (for multi-class task), numpy array, pandas Series, pandas DataFrame (for multi-class task) or None, optional (default=None)
```